Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Multistage docker build, requires docker 17.05
 
 # builder stage
-FROM ubuntu:20.04 as builder
+FROM ubuntu:latest as builder
```

Comment:
It sounds like a good idea for the version to be self-adjusting across time, but I'd prefer having the version fixed in stone, until there's somebody, who updates it to the next latest version, and collects the medal for fixing all the resulting incompatibilities.

Your proposed change would strike you when you're least expecting it.

Reply:
I agree, the base should be a known-good version.